src/fe-common/irc/fe-irc-server.c

```diff
@@ -50,12 +50,19 @@ const char *get_visible_target(IRC_SERVER_REC *server, const char *target)
 
 	return target;
 }
+
 /* SYNTAX: SERVER ADD [-4 | -6] [-ssl] [-ssl_cert <cert>] [-ssl_pkey <pkey>] [-ssl_pass <password>]
```

Comment:
Just recycle the existing comment block, the options that are accepted are the same for ADD and MODIFY, and that way we don't have to keep in-sync so many different versions of the same text.